reana/cli.py

```diff
@@ -456,13 +469,14 @@ def docker_build(user, tag, component, no_cache):
     """
     components = select_components(component)
     for component in components:
-        if no_cache:
-            cmd = 'docker build --no-cache -t {0}/{1}:{2} .'.format(
-                user, component, tag)
-        else:
-            cmd = 'docker build -t {0}/{1}:{2} .'.format(
-                user, component, tag)
-        run_command(cmd, component)
+        if is_component_dockerisable(component):
```

Comment:
Shall we warn the user if the component is not dockerised? (upstairs I wrote dockerized, sorry about that) If it is not, the answer is empty and it might drive people to think that something went wrong.

Something like: `'{0} does not contain a Dockerfile.'.format(component)`.